### DISCUSSION TITLE
Implicit to Explicit TimeIntegrator in the middle of a run

### DISCUSSION THREAD

**Discussion by maxnezdyur** [idaholab/moose#20179] [Q&A Modules: General] [answered]
Is there a way to switch from an implicit to an explicit time integrator in the middle of a simulation? I have a situation where I need to run a simulation to a specific time that I cannot shorten. Then at a particular time when I know the explicit solver would handle the problem better, I want to switch to it.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Currently the only option is to use the restart or recover system:
https://mooseframework.inl.gov/application_usage/restart_recover.html
You could also write your own time integrator.

Guillaume
